packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -270,30 +304,25 @@ class FlameChartNode<T> {
 
   Widget buildWidget(bool selected) {
     selected = selectable ? selected : false;
-    return Tooltip(
-      message: tooltip,
-      waitDuration: tooltipWait,
-      preferBelow: false,
-      child: GestureDetector(
-        onTap: () => onSelected(data),
-        child: Container(
-          width: rect.width,
-          height: rect.height,
-          padding: const EdgeInsets.symmetric(horizontal: 6.0),
-          alignment: Alignment.centerLeft,
-          color: selected ? _selectedNodeColor : backgroundColor,
-          child: rect.width > _minWidthForText
-              ? Text(
-                  text,
-                  textAlign: TextAlign.left,
-                  overflow: TextOverflow.ellipsis,
-                  style: TextStyle(
-                    color: selected ? Colors.black : textColor,
-                  ),
-                )
-              : const SizedBox(),
-        ),
-      ),
+    // TODO(kenz): figure out a way to add tooltips without crippling
```

Comment:
As a suggestion, we can have the top-level widget that's already doing gesture detection be responsible for generating the tooltips.  One mouseover of a particular item in a row, we'd update the tooltip text.

I believe the class to do this with is MouseRegion: https://api.flutter.dev/flutter/widgets/MouseRegion-class.html

This might take a bit of work on the tracking logic; feel free to consider this as out-of-scope for this PR.

Reply:
Added this info to the TODO.